src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -1357,7 +1362,7 @@ public String takeScreenshot(String basename) {
         return screenshotFile;
     }
 
-    private String getScreenshotLink(String screenshotFile) {
+    public String getScreenshotLink(String screenshotFile) {
```

Comment:
Protected would suffice. iirc I dit this because I couldn't figure out a good way to detect what driver we needed for the screenshot (see the very ugly stacktrace-check in seleniumHelper).